public/app/features/alerting/unified/Analytics.ts

```diff
@@ -8,3 +11,30 @@ export const LogMessages = {
   cancelSavingAlertRule: 'user canceled alert rule creation',
   successSavingAlertRule: 'alert rule saved successfully',
 };
+
+// logInfo from '@grafana/runtime' should be used, but it doesn't handle Grafana JS Agent and Sentry correctly
+export function logInfo(message: string, context: Record<string, string | number> = {}) {
+  if (config.grafanaJavascriptAgent.enabled) {
+    agent.api.pushLog([message], {
+      level: GrafanaLogLevel.INFO,
+      context: { ...context, module: 'Alerting' },
+    });
+  }
+}
+
+export function withPerfLogging<TFunc extends (...args: any[]) => Promise<any>>(
```

Comment:
@ashharrison90 I believe there is no better way to handle `any` types there (If there is, I'm happy to implement).  
Is there a way to bypass betterer check for this line?

Reply:
hey @konrad147, sorry for the late reply was at an offsite 😅 

given that the actual types when using this seem fine, i don't think there's a problem with the `any` here. the betterer rule is there to encourage us to think about these things, and 95% of the time we shouldn't need `any`, but think in this case it makes sense 😄 

you can ignore it via something like `// eslint-disable-next-line @typescript-eslint/no-explicit-any`

Reply:
Thanks!